src/app.js

```diff
@@ -16,26 +15,20 @@ import findOrCreateUser from './services/accessValidation';
 import logger, { requestLogger } from './logger';
 
 const app = express();
-const MemoryStore = memorystore(session);
+
 const oauth2CallbackPath = '/oauth2-client/login/oauth2/code/';
 
 app.use(requestLogger);
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(helmet());
-app.use(session({
-  secret: process.env.SESSION_SECRET,
-  key: 'session',
-  cookie: {
-    httpOnly: true,
-    sameSite: 'lax',
-  },
-  rolling: true,
-  store: new MemoryStore({ // Potentially change this to a different store
-    checkPeriod: 86400000, // prune expired entries every 24h
-  }),
-  saveUninitialized: false,
-  resave: false,
+
+app.use(cookieSession({
```

Comment:
I think setting the secure flag for sandbox/dev/staging/prod makes sense here.

Reply:
It is set to `secure `by default for `https`, so we would get it for production. We could change it for the lower environments; the thinking was that we wouldn't use the real user data in sandbox/dev/staging and not setting it would eliminate a need to set express behind proxies.
https://expressjs.com/en/guide/behind-proxies.html

Reply:
Ah, that makes sense.